modules/cnv-labels-nodes-workloadscheduling.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_advanced_vm_management/cnv-dedicated-resources-vm.adoc
+// * cnv/cnv_vm_templates/cnv-dedicated-resources-vm-template.adoc
+
+[id="cnv-labels-nodes-workloadscheduling_{context}"]
+
+= Preparing to enable dedicated resources
+
+Prepare to enable dedicated resources by doing the following.
+
+.Procedure
+* Ensure that the `cpumanager` label is `true` for each node on which you want
+to schedule virtual machines with dedicated resources.
+** Verify that CPU Manager is running on the CPU node whose
```

Comment:
How?

Reply:
@aburdenthehand I have a question into Tomas and Radim about this. Chatted with Radim earlier and prepped him. Copied you.

Reply:
Update: a draft of this has been added to the preparation procedure. Radim and Tomas will review (sigh). Pan is asking Simone about any preconfiguration on the CNV Install front that may preclude the draft instructions I just entered. Will let you know and update here.